### PR TITLE
 Add more test for remap without shared file system setting

### PR DESCRIPTION
**Pull request by classicsong**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/end2end-tests/graphstorm-ec/mgpu_test.sh [hunk 1]
```diff
@@ -156,6 +156,14 @@ python3 check_infer.py --train_embout /data/gsgnn_ec/emb/ --infer_embout /data/g
 
 error_and_exit $?
 
+python3 -m graphstorm.run.gs_edge_classification --inference --workspace $GS_HOME/inference_scripts/ep_infer --num-trainers $NUM_INFO_TRAINERS --num-servers 1 --num-samplers 0 --part-config /data/movielen_100k_multi_label_ec/movie-lens-100k.json --ip-config ip_list.txt --ssh-port 2222 --cf ml_ec_infer.yaml  --multilabel true --num-classes 6 --node-feat-name movie:title user:feat --use-mini-batch-infer false --save-embed-path /data/gsgnn_ec/infer-emb/ --restore-model-path /data/gsgnn_ec/epoch-$best_epoch/ --save-prediction-path /data/gsgnn_ec/prediction-no-sf/ --logging-file /tmp/log.txt  --logging-level debug --with-shared-fs False
+
+error_and_exit $?
+
+python3 $GS_HOME/tests/end2end-tests/check_predict_result.py --infer-prediction /data/gsgnn_ec/prediction/user_rating_movie/ --no-sfs-prediction /data/gsgnn_ec/prediction-no-sf/user_rating_movie/ --edge-prediction
```
Comment (zheng-da) [inline tests/end2end-tests/graphstorm-ec/mgpu_test.sh:163]: i don't quite understand. how do you simulate no shared filesystem?

Comment (classicsong) [reply tests/end2end-tests/graphstorm-ec/mgpu_test.sh:163]: > i don't quite understand. how do you simulate no shared filesystem?

When explicitly set --with-shared-fs False, the remap command will run code in no-shared-fs branch.

Comment (zheng-da) [reply tests/end2end-tests/graphstorm-ec/mgpu_test.sh:163]: but this is not really a distributed environment. that means the folder still contains all partitions.

Comment (classicsong) [reply tests/end2end-tests/graphstorm-ec/mgpu_test.sh:163]: There are two separate tests for this remap with no-shared-fs.
One is here: https://github.com/awslabs/graphstorm/blob/id-remap-dev/tests/end2end-tests/data_process/test.sh#L179-L208. It only tests the remap command. I manually build two folders to mimic the distributed environment without shared file system.

The tests here are testing whether the launch script can work with the remap command with no-shared-fs.

**Review by zheng-da — APPROVED** [review]

**Pull request merged**